setup.sh

```diff
@@ -295,7 +295,7 @@ if [[ -z "$SKIP_AUTH" ]]; then
     read -rp "Would you like to configure $(tput bold)$(tput setaf 3)end-user authentication?$(tput sgr0) (y/n) " auth_yesno
 
     if [[ ${auth_yesno} == "y" ]]; then
-        sh ./scripts/configure_auth.sh
+        ./scripts/configure_auth.sh
```

Comment:
Is this the right way to run it or should we use `bash` explicitly?

Reply:
this should work just fine 👍